orchestrator/websocket/__init__.py

```diff
@@ -81,8 +80,10 @@ def init_websocket_manager(settings: AppSettings) -> WebSocketManager:
     return websocket_manager
 
 
-def create_process_websocket_data(process: ProcessTable, pStat: ProcessStat) -> dict:
-    return {"process": enrich_process(process, pStat)}
+def create_process_websocket_data(process_id: UUID) -> dict:
+    process = _get_process(process_id)
+    p_stat = load_process(process)
+    return {"process": enrich_process(process, p_stat)}
```

Comment:
@tjeerddie What do you think, can we delete this?

If I'm not mistaken, this was exclusively used to broadcast the entire process data to the V1 GUI; as opposed to now that we're only sending the process ID to the V2 GUI.

And we've already dropped support for the V1 GUI in a previous core release..

Reply:
that is true, with dropped V1 this can be removed.

Reply:
Aaand it's gone

Reply:
🔥